server/app.js

```diff
@@ -50,6 +50,7 @@ app.use("/auth", authRouter);
 app.use("/users", userRouter);
 app.use("/request", requestRouter);
 app.use("/profile", profileRouter);
+app.use("/availability", availabilityRouter);
```

Comment:
Since the availability is a part of the Profile, I don't think we need a separate router for availability. Wouldn't it be better to include it under the `/profile` route? :)